cpp/ql/src/semmle/code/cpp/security/Overflow.qll

```diff
@@ -1,19 +1,25 @@
+/**
+ * Provides predicates for reasoning about when the value of an expression is
+ * guarded by an operation such as `<`, which confines its range.
+ */
+
 import cpp
 import semmle.code.cpp.controlflow.Dominance
 
-/*
- * Guarding
+/**
+ * Holds if the value of `use` is guarded using `abs`.
  */
-
-/** is the size of this use guarded using 'abs'? */
 predicate guardedAbs(Operation e, Expr use) {
   exists(FunctionCall fc | fc.getTarget().getName() = "abs" |
     fc.getArgument(0).getAChild*() = use and
     guardedLesser(e, fc)
   )
 }
 
-/** This is `BasicBlock.getNode`, restricted to `Stmt` for performance. */
+/**
+ * Gets the result of `BasicBlock.getNode`, but only on a `Stmt` (for
```

Comment:
The predicate doesn't actually get that result. The arguments are flipped, so the result of `getNode` is an argument of this predicate.

Reply:
Ah, I trusted the original QLDoc too much.  Well spotted.